index.html

```diff
@@ -33,6 +33,9 @@ <h2 class="section-header">About Me</h2>
            Looking forward to connect and be a part of growth collabration!
       </p>
 
+      <p>
+          I have taken various computer programing courses one of which was a course on how to use the git framework to work collaboratively on a team. I have also taken classes in java, c, ocaml, ruby, go, intercal, forth, coq. I also familiar with Android and django platforms. 
```

Comment:
The second sentence needs to be changed as the information is not accurate.   Go, intercal, forth, coq, and django platform need to be removed.